requests/requests.py

```diff
@@ -0,0 +1,63 @@
+TOKEN = ''
```

Comment:
Your pull request: REQUESTS-1021 Added verification.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']

Your commit: VERIFICATION-1021 Added verification.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: REQUESTS-1021 Added requests.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']